.github/workflows/issue-mark-duplicate.yml

```diff
@@ -0,0 +1,18 @@
+name: Issue Mark Duplicate
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  mark-duplicate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: mark-duplicate
+        uses: actions-cool/issues-helper@v1.6
+        with:
+          actions: 'mark-duplicate'
+          token: ${{ secrets.GITHUB_TOKEN }}
+          duplicate-command: '/d'
```

Comment:
Since these are user-facing commands we shouldn't use shorthands. The existing "Duplicate of" which is also understood by GitHub (and potentially does the same thing at some point) seems sufficient to me

Reply:
OK, I will remove it. So just `Duplicate of` can use.

when use `labels` it will remove other labels. 

Now use `duplicate-labels` it will additional label, will not affect the original